arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -707,12 +711,6 @@
       "x-ms-external": true
     },
     "RedisCreateProperties" : {
-      "properties": {
-        "sku": {
-          "$ref": "#/definitions/Sku",
-          "description": "The SKU of the Redis cache to deploy."
-        }
-      },
       "required": [
```

Comment:
You can't change a property from an `allOf` composition to `required` . 
(The property won't actually get marked as required at all.)